rosette/src/Console.cc

```diff
@@ -52,33 +42,27 @@
 
 #ifdef MIPS_SGI_SYSV
```

Comment:
Do we care about MIPS_SGI_SYSV at all?

Reply:
Not even a little. An upcoming PR will strip all the "portability" stuff that's not needed anymore.

Reply:
Good to hear.